pkg/controller/healthz.go

```diff
@@ -20,23 +20,23 @@ import (
 	"net/http"
 	"time"
 
+	"github.com/google/exposure-notifications-server/pkg/logging"
 	"github.com/google/exposure-notifications-verification-server/pkg/database"
 	"github.com/google/exposure-notifications-verification-server/pkg/render"
 
-	"github.com/google/exposure-notifications-server/pkg/logging"
-
 	"golang.org/x/time/rate"
 )
 
 var (
 	rl = rate.NewLimiter(rate.Every(time.Minute), 1)
 )
 
-func HandleHealthz(hctx context.Context, cfg *database.Config, h *render.Renderer) http.Handler {
-	logger := logging.FromContext(hctx).Named("healthz")
-
+func HandleHealthz(ctx context.Context, cfg *database.Config, h *render.Renderer) http.Handler {
```

Comment:
The `ctx` here is not actually used right? In the body of the returned function, `ctx` is from `r.Context()`.

Reply:
Yea, removed. Still confused about this data race though...